### PR TITLE
Map - Allow enabling BFT setting after mission start

### PR DESCRIPTION
**Pull request by shukari**
**When merged this pull request will:**
- change to CBA_fnc_addSetting not the internal fnc CBA_settings_fnc_init
- some vars can edited in the mission
- if BFT was disabled at mission start an den enabled it now works...

### CODE REVIEW

**Comment by commy2** [comment]
Can you also disable it on the fly?

**Comment by shukari** [comment]
@commy2 disable was always possible because if false:
https://github.com/acemod/ACE3/blob/fabe625d815fa99720144cfcabd2e4042db4f8f3/addons/map/functions/fnc_blueForceTrackingUpdate.sqf#L27

File: addons/map/initSettings.sqf [hunk 1]
```diff
@@ -66,9 +66,17 @@
...
     false,
     true,
-    {[QGVAR(BFT_Enabled), _this] call EFUNC(common,cbaSettings_settingChanged)},
-    true // Needs mission restart
-] call CBA_settings_fnc_init;
+    {
+        [QGVAR(BFT_Enabled), _this] call EFUNC(common,cbaSettings_settingChanged);
+        
+        if (isNil QGVAR(BFT_markers)) then
+        {
```
Comment (synixebrett) [inline addons/map/initSettings.sqf]: Brackets should be on the same line as the `if` statement

**Comment by commy2** [comment]
[This](https://github.com/acemod/ACE3/blob/master/addons/map/XEH_postInitClient.sqf#L27-L31) is not superfluous and should be deleted.

File: addons/map/initSettings.sqf [hunk 2]
```diff
@@ -78,8 +85,8 @@
     [0, 30, 1, 1],
     true,
     {[QGVAR(BFT_Interval), _this] call EFUNC(common,cbaSettings_settingChanged)},
-    true // Needs mission restart
-] call CBA_settings_fnc_init;
+    false
```
Comment (PabstMirror) [inline addons/map/initSettings.sqf]: once pfeh is running, there is no way to change this
```suggestion
    true // Needs mission restart
```

**Pull request merged**